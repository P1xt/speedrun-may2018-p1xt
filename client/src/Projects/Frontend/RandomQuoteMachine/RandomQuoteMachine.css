.RandomQuoteMachine {
  text-align: left;
  margin-top: 40px; }
  .RandomQuoteMachine > h1 {
    text-transform: none; }
