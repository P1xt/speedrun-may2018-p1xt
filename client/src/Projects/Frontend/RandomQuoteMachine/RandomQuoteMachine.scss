.RandomQuoteMachine {
  text-align: left;
  margin-top: 40px;

  &>h1 {
    text-transform: none
  }
  >.card {
    max-width: 400px;
    width: 400px;
    margin: 10px auto;
    display: block;
    .card-text {
      min-height: 120px;
    }
  }
  &_buttons {
    display: flex;
    flex-direction: row;
    justify-content: space-between;
  }
}
