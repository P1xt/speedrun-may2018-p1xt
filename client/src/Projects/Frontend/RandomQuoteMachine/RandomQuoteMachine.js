import React from "react";
import { Container, Row, Col } from 'reactstrap';

import "./RandomQuoteMachine.css";

const RandomQuoteMachine = () => {
  return (
    <div className="RandomQuoteMachine">
      <Container>
        <Row>
          <Col>
            <h1>Random Quote Machine</h1>
          </Col>
        </Row>
      </Container>
    </div>
  );
};

export default RandomQuoteMachine;
