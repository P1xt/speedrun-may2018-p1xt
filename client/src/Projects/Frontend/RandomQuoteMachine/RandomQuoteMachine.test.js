import React from "react";
import renderer from 'react-test-renderer';
import { shallow } from "enzyme";

import RandomQuoteMachine from "./RandomQuoteMachine";

describe("<RandomQuoteMachine />", () => {
  it("renders without crashing", () => {
    shallow(<RandomQuoteMachine />);
  });
  it("renders correctly", () => {
    const tree = renderer.create(<RandomQuoteMachine />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
