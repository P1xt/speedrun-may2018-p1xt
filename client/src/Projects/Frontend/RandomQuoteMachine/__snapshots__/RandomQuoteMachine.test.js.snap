// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RandomQuoteMachine /> renders correctly 1`] = `
<div
  className="RandomQuoteMachine"
>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="col"
      >
        <h1>
          Random Quote Machine
        </h1>
      </div>
    </div>
  </div>
</div>
`;
